src/rgw/rgw_admin.cc

```diff
@@ -478,6 +480,7 @@ enum {
   OPT_ZONEGROUP_PLACEMENT_MODIFY,
   OPT_ZONEGROUP_PLACEMENT_RM,
   OPT_ZONEGROUP_PLACEMENT_LIST,
+  OPT_ZONEGROUP_PLACEMENT_GET,
```

Comment:
can you please add OPT_ZONEGROUP_PLACEMENT_GET and OPT_ZONE_PLACEMENT_GET to both of `raw_storage_ops_list` and `readonly_ops_list`?

Reply:
Thank you for your suggestion. OPT_ZONEGROUP_PLACEMENT_GET and OPT_ZONE_PLACEMENT_GET have been added to raw_storage_ops_list and readonly_ops_list.